src/initial_corpus/mod.rs

```diff
@@ -29,12 +29,23 @@ pub fn load_starting_corpus(
 ) -> Result<Vec<OpenApiInput>, Box<dyn std::error::Error>> {
     let mut corpus_vec = vec![];
     for file in fs::read_dir(corpus_dir)? {
-        let file = file?.path();
-        match serde_yaml::from_reader(std::fs::File::open(file)?) {
-            Ok(input) => corpus_vec.push(input),
-            Err(err) => return Err(err.into()),
+        let file = file?;
+        let file_name = file.file_name();
+        let file_name_str = file_name.to_str().unwrap();
```

Comment:
This will panic for invalid utf-8. I propose to at least give a semi-helpful error message.

```suggestion
        let file_name_str = file_name.to_str().expect("Invalid utf-8 encountered in filenames in the corpus directory");
```

But maybe better is to ask for an OsStr, check if it starts with `'.'` using the `[u8]` representation from `as_encoded_bytes`, and print it using the Debug trait `{file_name_os_str:?}` (supposedly the OS can print its own strings but hey who knows).